tests/python/pants_test/goal/test_run_tracker.py

```diff
@@ -59,6 +59,11 @@ def mk_url(path):
     server.shutdown()
     server.server_close()
 
+  def test_upload_stats(self):
+    self.assert_upload_stats(response_code=200)
+    # self.assert_upload_stats(response_code=201)
```

Comment:
I've re-started failing shards. Perhaps these can be uncommented now? If the failures are timeouts, this seems to be a general issue. Stu sent an email about adjusting timeouts or marking flaky, but, fwict, no one has done much of this and instead lots of PRs are being submitted when the only failures are timeouts.